my-ansible-playbook.yml

```diff
@@ -0,0 +1,69 @@
+- name: set up to do app
+  hosts: webservers
+  remote_user: ec2-user
+  vars_prompt:
+    - name: trello_key
+      prompt: 'what is your trello key'
+      private: no
+      default: ''
+    - name: trello_token
+      prompt: 'what is your trello token'
+      private: no
```

Comment:
This is fine here, but generally we will want to treat this token as private; so e.g. if this was running in a pipeline then we'd want to avoid this being accidentally logged to the console